### PR TITLE
Disable CoreDNS K8scc templates

### PR DESCRIPTION
**Pull request by fgimenez**
Towards https://github.com/giantswarm/giantswarm/issues/1902

### CODE REVIEW

**Comment by fgimenez** [comment]
Good catch, I've added an entry similar to what we have in `v15` for `nginx-ingress-controller`, PTAL

File: service/controller/v17/version_bundle.go [hunk 1]
```diff
@@ -12,6 +12,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Add security group descriptions for master nodes.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "cloudconfig",
+				Description: "Removed CoreDNS related components (will be managed by chart-operator).",
```
Comment (kopiczko) [inline service/controller/v17/version_bundle.go]: Nit: I would skip `will be`. They should be already managed at the time we release this version.

Comment (rossf7) [reply service/controller/v17/version_bundle.go]: To me its not clear without the will be but `(now managed by chart-operator)' is shorter and clear.

Comment (kopiczko) [reply service/controller/v17/version_bundle.go]: As I see it's only done in v15.

Comment (fgimenez) [reply service/controller/v17/version_bundle.go]: Good catch, fixed.

**Comment by fgimenez** [comment]
For the record this is not working, when we remove the k8scc CoreDNS manifests Calico networking fails. See https://github.com/giantswarm/giantswarm/issues/4242 for details.

**Comment by fgimenez** [comment]
Tested on ginger.

**Pull request merged**